### PR TITLE
CAPI: Add more digitalocean images

### PR DESCRIPTION
**Pull request by prksu**
This PR adds more Cluster API build images for DigitalOcean.

/cc @detiber

### CODE REVIEW

**Review by xmudrii — APPROVED** [review]
/lgtm

File: images/capi/packer/digitalocean/centos-7.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "build_name": "centos-7",
+  "source_image": "centos-7-x64",
+  "snapshot_name_suffix": "on CentOS 7"
+}
```
Comment (cpanato) [inline images/capi/packer/digitalocean/centos-7.json:5]: nit: new line

File: images/capi/packer/digitalocean/packer.json [hunk 2]
```diff
@@ -82,4 +74,4 @@
       ]
     }
   ]
-}
+}
```
Comment (cpanato) [inline images/capi/packer/digitalocean/packer.json:77]: nit: new line

File: images/capi/packer/digitalocean/ubuntu-2004.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (cpanato) [inline images/capi/packer/digitalocean/ubuntu-2004.json:1]: nit: new line

File: images/capi/packer/digitalocean/ubuntu-1804.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "build_name": "ubuntu-1804",
+  "source_image": "ubuntu-18-04-x64",
+  "snapshot_name_suffix": "on Ubuntu 18.04"
+}
```
Comment (cpanato) [inline images/capi/packer/digitalocean/ubuntu-1804.json:5]: nit: new line

File: images/capi/packer/digitalocean/centos-7.json (hunk 1 above)
Comment (prksu) [reply images/capi/packer/digitalocean/centos-7.json]: fixed, thanks!

File: images/capi/packer/digitalocean/packer.json (hunk 2 above)
Comment (prksu) [reply images/capi/packer/digitalocean/packer.json]: fixed, thanks!

File: images/capi/packer/digitalocean/ubuntu-2004.json (hunk 3 above)
Comment (prksu) [reply images/capi/packer/digitalocean/ubuntu-2004.json:1]: fixed, thanks!

File: images/capi/packer/digitalocean/ubuntu-1804.json (hunk 4 above)
Comment (prksu) [reply images/capi/packer/digitalocean/ubuntu-1804.json]: fixed, thanks!

**Review by cpanato — APPROVED** [review]
/lgtm

**Pull request merged**